components/manager-ip-setter/scripts/manager-ip-setter.sh

```diff
@@ -35,11 +35,13 @@ function set_manager_ip() {
   echo "Restarting services.."
   # Restarting all (except postgres) to avoid issues with not correctly reloading SSL certs
   systemctl restart nginx
-  systemctl restart cloudify-rabbitmq
   systemctl restart cloudify-amqpinflux
   systemctl restart cloudify-influxdb
-  systemctl restart cloudify-restservice
   systemctl restart cloudify-mgmtworker
+  systemctl restart cloudify-rabbitmq
+  systemctl restart cloudify-restservice
+  systemctl restart cloudify-riemann
+  systemctl restart cloudify-stage
```

Comment:
I just put them all in to save on any present or future issues related to any files we change (e.g. if stage was reading and caching the internal public cert or any IPs then this would just make that problem go away before it became an issue).